Gemfile

```diff
@@ -8,6 +8,13 @@ group :development do
   gem 'pry'
 end
 
+if RUBY_VERSION < "2.0"
+  gem 'json', '< 2.0.0'
+  gem 'net-ssh', '<= 2.10'
+  gem 'rest-client', '< 1.7'
```

Comment:
This has nasty implications...http://projects.theforeman.org/issues/13698 can we get 1.8.x at least? https://github.com/rest-client/rest-client/blob/1.8.x/rest-client.gemspec